login/app/login/login-view-model.js

```diff
@@ -6,7 +6,10 @@ function LoginViewModel() {
     viewModel.email = "";
     viewModel.password = "";
 
-    viewModel.onSignIn = function () {
+    viewModel.onSignInButtonTap = function () {
```

Comment:
I find it more intuitive if all "event" handlers are defined in the page code-behind and not in the view model (they may call view model methods though). A bit more verbose but otherwise I keep wondering why we have "tap" methods here and there.